app/core/core-ui/src/main/kotlin/com/hedvig/android/core/ui/text/WarningTextWithIcon.kt

```diff
@@ -26,3 +30,21 @@ fun WarningTextWithIcon(text: String, modifier: Modifier = Modifier) {
     Text(text)
   }
 }
+
+@Composable
+fun WarningTextWithIconForInput(text: String) {
```

Comment:
Looks like quite a specific component. Not sure if it could maybe just live in the module that uses it?
I kinda feel the same about the `WarningTextWithIcon` above, just from the name I am not 100% sure what it does or where it's used.

In any case, even if you move it or not, such components that are in core-ui or design-system always benefit by also taking in a `modifier` so that they can be modified as needed.